lib/plug/conn.ex

```diff
@@ -257,6 +257,16 @@ defmodule Plug.Conn do
     """
   end
 
+  defmodule InvalidStatusError do
```

Comment:
I don't think we need to introduce a new exception. I would just use ArgumentError.

    raise ArgumentError, "inform expects a status code between 100 and 199, got: ..."